src/Builders/PaymayaCheckoutBuilder.php

```diff
@@ -139,6 +140,10 @@ public function getItems(): array
      */
     public function setItem(PaymayaItem $paymayaItem, $quantity = 1): PaymayaCheckoutBuilder
     {
+        if (!$paymayaItem instanceof Model) {
+            throw new \ErrorException(get_class($paymayaItem).' must subclass of '.Model::class);
```

Comment:
Can you change to ".. must be a subclass of "...

Reply:
Done!